scripts/ErrorReporter/error_dialog_app.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import argparse
+
+parser = argparse.ArgumentParser(description='Pass in exit_code')
+parser.add_argument('--exitcode', dest='exit_code')
+parser.add_argument('--directory', dest='directory')
+command_line_args = parser.parse_args()
+
+sys.path.insert(0, command_line_args.directory)
+
+from PyQt4 import QtGui # noqa
+
+import mantid # noqa
+from ErrorReporter import resources # noqa
+from mantid.kernel import UsageService # noqa
+from ErrorReporter.error_report_presenter import ErrorReporterPresenter # noqa
+from ErrorReporter.errorreport import CrashReportPage # noqa
+
+
+def main():
+    if not UsageService.isEnabled():
```

Comment:
There's a minor issue here that if you have usage reports disabled and the crash happens the exit code of the process is `0`. I think we need to make sure main always returns an integer and then main gets called as `sys.exit(main())` from below.